pkg/install/cleanup_unix.go

```diff
@@ -206,6 +207,33 @@ func (c *CleanUpConfig) RemoveAllDirectories() error {
 		fmtError := fmt.Errorf("failed to delete %v. err: %v", c.runDir, err)
 		msg = append(msg, fmtError.Error())
 	}
+	if len(msg) > 0 {
+		return fmt.Errorf("%v", strings.Join(msg, ", "))
+	}
+	return removeCNILeftovers()
+}
+
+// remove calico and kuberouter leftover configuration files
+func removeCNILeftovers() error {
+	var msg []string
+
+	calico10Conflist := "/etc/cni/net.d/10-calico.conflist"
+	calicoKubeconfig := "/etc/cni/net.d/calico-kubeconfig"
+	kuberouterConf := "/etc/cni/net.d/10-kuberouter.conflist"
+
+	if err := os.Remove(calico10Conflist); err != nil {
+		fmtError := fmt.Errorf("failed to delete %v. err: %v", calico10Conflist, err)
+		msg = append(msg, fmtError.Error())
```

Comment:
If the cluster was running kube-router CNI, this file will not exist and thus `os.Remove` will fail with:
```
remove /etc/cni/net.d/10-calico.conflist: no such file or directory
```

So trying to remove a non-existing file should not be an error in reset. Same goes for all the files of course

Reply:
It doesn't break the command. It just returns this info at the end
```
INFO[2021-04-08 10:17:27] deleting k0s generated data-dir (/var/lib/k0s) and run-dir (/run/k0s) 
INFO failed to delete /etc/cni/net.d/10-calico.conflist. err: remove /etc/cni/net.d/10-calico.conflist: no such file or directory, failed to delete /etc/cni/net.d/calico-kubeconfig. err: remove /etc/cni/net.d/calico-kubeconfig: no such file or directory 
INFO[2021-04-08 10:17:27] k0s cleanup operations done. To ensure a full reset, a node reboot is recommended. 
```

Reply:
which is indeed confusing

Reply:
only debug will print what was not deleted:
```
# k0s reset --debug
...
DEBU[2021-04-08 10:37:35] failed to delete /etc/cni/net.d/10-calico.conflist. err: remove /etc/cni/net.d/10-calico.conflist: no such file or directory, failed to delete /etc/cni/net.d/calico-kubeconfig. err: remove /etc/cni/net.d/calico-kubeconfig: no such file or directory, failed to delete /etc/cni/net.d/10-kuberouter.conflist. err: remove /etc/cni/net.d/10-kuberouter.conflist: no such file or directory 
```